src/fstext/lattice-weight.h

```diff
@@ -592,6 +620,34 @@ class NaturalLess<CompactLatticeWeightTpl<LatticeWeightTpl<FloatType>, IntType>
     return (Compare(w1, w2) == 1);
   }
 };
+template<>
+class NaturalLess<CompactLatticeWeightTpl<LatticeWeightTpl<float>, int> > {
```

Comment:
Should the `int` be `int32_t` for consistency? I think `int` is not generally used in the codebase.

Reply:
yes, this should be int32 (which is what we call it Kaldi code).